packages/jaeger-ui/src/components/DependencyGraph/DAG.tsx

```diff
@@ -104,35 +104,56 @@ export default class DAG extends React.Component<TProps> {
     this.layoutManager.stopAndRelease();
   }
 
+  Controls = () => {
```

Comment:
this doesn't seem the right place. Plexus has a MiniMap module that deals with scaling and has a Reset button, I think the extra buttons should go there and apply automatically everywhere

Reply:
Actually the Wrapper I have used requires the Control buttons to be placed outside the Component you want to Transform, and since the graph is being transformed the controls can't be placed inside the graph.

Reply:
It doesn't mean these wrappers cannot be part of the Digraph component itself

Reply:
I have wrapped the Trace-Diff Graph Div and DAG Graph Div with the TransforWrapper which allows the whole div to be zoomed in and out.
Do you want me to just wrap the graph instead, I am not sure about that can you kindly explain on the specific component that will help me a lot!

Reply:
I am suggesting to make these changes inside the Plexus package. This is generic functionality, it does not make sense to re-add it in every usage site.

Reply:
Got it, will do that. I am currently on a vacation for a week. So, please excuse me for a while and I will get back to it!

Reply:
Hi @yurishkuro, I will start working on the changes you mentioned now.
Should I continue making changes and commits in this PR or should I close this and open an entirely new PR from another branch?